### PR TITLE
Calibration processor and deserializer

### PR DESCRIPTION
**Pull request by RaphaelAcademia**
Testing _scale_sensors_ in _calibration_processor.py_:
```
print(scale_sensors(100,0,1023,0, 512)) # outputs -50.048875855327466
print(scale_sensors(100,1023,1023,0, 512)) # outputs 49.951124144672534
```

Testing _get_state_based_on_accel_ in _state_lookup.py__:
```
print(get_state_based_on_accel(json.loads('{"x": 7, "y": 15, "z": 15}'))) # returns START
print(get_state_based_on_accel(json.loads('{"x": 7, "y": 23, "z": 15}'))) # returns MOVE_RIGHT
```

### CODE REVIEW

File: software/drivers/state_lookup.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import json
+
+def get_states_from_json():
+    with open("states.json", "r") as f:
```
Comment (RaphaelAcademia) [inline software/drivers/state_lookup.py:4]: should we use a config file or something similar to specify file paths?

**Comment by RaphaelAcademia** [comment]
**TODO:** Change values in JSON file to actual values once we get them

File: software/drivers/state_lookup.py (hunk 1 above)
Comment (bricktrooper) [reply software/drivers/state_lookup.py:4]: Yeah that would be useful.  You could also pass it as a command line arg, but having a config file makes more sense for the GUI.

File: software/drivers/states.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
+{
```
Comment (bricktrooper) [inline software/drivers/states.json:1]: What does the file specify?  the min and max for specific states?

**Pull request merged**